SamMou.Api/Startup.cs

```diff
@@ -1,25 +1,19 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 using SamMou.Api.DataContext;
 using SamMou.Api.Services;
 using SamMou.Api.Services.Interface;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.HttpsPolicy;
-using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
 using Microsoft.EntityFrameworkCore;
-using SamMou.Api.GraphQL;
-using SamMou.Api.Models;
 using GraphQL.Utilities;
-using GraphQL.EntityFramework;
+using Microsoft.Extensions.Logging;
+using GraphQL.Server;
+using SamMou.Api.GraphQL;
 using GraphQL;
-using GraphQL.Types;
+using GraphQL.SystemTextJson;
+using Microsoft.AspNetCore.Http;
```

Comment:
I think it's not sorted, can we sort this?

Reply:
okiee